packages/mobile/src/fiatExchanges/utils.tsx

```diff
@@ -130,3 +149,20 @@ export function getProviderAvailability(
   }
   return features
 }
+
+// Leaving unoptimized for now because sorting is most relevant when fees will be visible
+export const sortProviders = (provider1: Provider, provider2: Provider) => {
```

Comment:
This function is currently not argument order-agnostic. i.e `sortProviders(A, B)` could return 1 and `sortProviders(B, A)` could return 1 as well, which wouldn't make sense.

If both providers are unavailable or both are restricted I think we should return 0.

Reply:
Yep that's true. I didn't bother to put that in because specific order doesn't really matter...all we care about is that it shows:
1. available and unrestricted providers
2. restricted providers
3. unavailable providers

in that order. The more interesting sorting will come when we have fee information and can sort based on that!